src/schematics/src/update/update-0.13.0.ts

```diff
@@ -76,6 +76,16 @@ Start changes for Clarity 0.13.0
             );
           }
 
+          // Alert to using clrType in hide/show panel in datagrid (https://github.com/vmware/clarity/pull/2546)
+          if (updated.search(/clrType/g) > -1) {
+            logMessage(
+              'error',
+              'ACTION REQUIRED: Clarity has removed clrType from the column hide/show panel in datagrid, please remove the attribute from your template.',
```

Comment:
The reality is that we've removed `clrType` _and_ the OK button variant. That means not just removing the attribute, but also removing the OK button.